pytest-backend/pyproject.toml

```diff
@@ -46,7 +52,7 @@ force_grid_wrap = 2
 
 
 [tool.pylint.master]
-fail-under = 8.1
+fail-under = 6.3
```

Comment:
What changed that this went down?

Reply:
Not sure about the long-term trends, but until this PR, this linting score wasn't being tested in the CI, so it could be 1) that it was never that high or 2) that it drifted without CI or pre-commit checks. :grimacing:

Reply:
Never that high is likely, thx